JsonPatch/JsonPatch.cs

```diff
@@ -95,7 +104,7 @@ public class PatchJsonConverter : JsonConverter<JsonPatch>
 	/// <returns>The converted value.</returns>
 	public override JsonPatch Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 	{
-		var operations = JsonSerializer.Deserialize<List<PatchOperation>>(ref reader, new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase })!;
```

Comment:
This is where we lost the CamelCase (and presumably the case insensitivity) from before. You can add it back through the JsonSourceGeneratorOptions attribute on the PatchSerializerContext if that makes sense? But I also saw you did name aliases on the Model objects, that might be cleaner.